### ISSUE TITLE
Select multiple delete option

### ISSUE THREAD

**Issue by ve3elb** [jekhokie/raspberry-noaa-v2#447]
In raspberry noaa can a select option be added to the admin tab of the page for deleting more than one capture or pass conflict at the same time. Well mostly for the captures deleting one at a time is a pain. Let me know what you think. Thank You

**Comment by dom-robinson** [comment]
i'd agree - nice one for a UI coder if we have one in the community? (even an explanation of a simple model to make this change would help - can't be difficult code, but it needs a simple 'way'...)

**Comment by MihajloPi** [comment]
Yes, this needs working on, but sadly, I don't really know web-dev that much and need some web developer.
